pages/docs/user-docs/docs-create-an-image.md

```diff
@@ -22,105 +23,136 @@ Binding image to loop
+If you have already created an image and wish to overwrite it, you can do so with the `--force` option.  This option must also follow the `create` sub-command.
+
+```
+$ singularity create --size 512 --force container2.img
+Initializing Singularity image subsystem
+Opening image file: container2.img
+Creating 512MiB image
+Binding image to loop
+Creating file system within image
+Image is done: container2.img
 
+$ ls -lh container*.img 
+-rwxr-xr-x 1 user group 513M Apr 15 11:39 container2.img
+-rwxr-xr-x 1 user group 769M Apr 15 11:11 container.img
+```
 
-## Mounting an image
-Once the image has been created, you can mount it with the following command:
+{% include asciicast.html source='docs-create-create.js' title='How to create images' author='davidgodlove@gmail.com'%}
 
-```bash
-$singularity mount container.img /mnt
+## Increasing the size of an existing image
+You can increase the size of an image after it has been instantiated by using the 'expand' Singularity sub-command as follows:
 
-Mounting image 'container.img' into '/mnt'
 ```
+$ ls -lh container.img 
+-rwxr-xr-x 1 user group 769M Apr 15 11:11 container.img
 
-To unmount, simply exit this new shell.
+$ singularity expand container.img 
+Initializing Singularity image subsystem
+Opening image file: container.img
+Expanding image by 768MiB
+Binding image to loop
+Checking file system
+e2fsck 1.42.13 (17-May-2015)
+Pass 1: Checking inodes, blocks, and sizes
+Pass 2: Checking directory structure
+Pass 3: Checking directory connectivity
+Pass 4: Checking reference counts
+Pass 5: Checking group summary information
+/dev/loop0: 11/49152 files (0.0% non-contiguous), 7387/196608 blocks
+Resizing file system
+resize2fs 1.42.13 (17-May-2015)
+Resizing the filesystem on /dev/loop0 to 393216 (4k) blocks.
+The filesystem on /dev/loop0 is now 393216 (4k) blocks long.
 
-```bash
-Singularity: \w> df -h /mnt
-Filesystem      Size  Used Avail Use% Mounted on
-/dev/loop1      740M  143M  559M  21% /mnt
+Image is done: container.img
+
+$ ls -lh container.img 
+-rwxr-xr-x 1 user group 1.6G Apr 15 12:21 container.img
 ```
- 
-When mounting the image in this manner, Singularity makes use of name-spaces such that the mount is only visible and accessible from within the current shell that Singularity will spawn. When you are finished, you can simply exit the shell and the file system will be automatically unmounted.
 
+Similar to the create sub-command, you can override the default size increase (which is 768MiB) by using the `--size` option.
 
-## Increasing the size of an existing image
-You can increase the size of an image after it has been instantiated by using the 'expand' Singularity sub-command as follows:
+{% include asciicast.html source='docs-create-expand.js' title='How to expand images' author='davidgodlove@gmail.com'%}
+
+## Mounting an image
+Once the image has been created, you can mount it to an existing directory like so:
```

Comment:
I think we need to add a quick rationale here for why a user would want to mount, because I was confused for a while and it's not readily apparent. Maybe something like:

While it's easy to shell into an image that is finished:

```
singularity shell container.img
```

or do the same with `--writable` to make changes:

```
sudo singularity shell --writable container.img
```
it may be the case that you want to view the container in a file browser. In this case, you would want to use the `mount` command to get direct access to the file system, and you can do this even if the image has not been completed yet (and shell would not work). This is a handy tool for development, or general inspection of your image.